tests/test_setup_profile.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+from unittest.mock import patch
+
+import setup_profile
+
+MOCK_PUBLIC_KEY = 'this is not a real rsa public key'
+MOCK_PRIVATE_KEY = 'this is not a real rsa private key'
+PUBLIC_KEYFILE = 'id_rsa.pub'
+PRIVATE_KEYFILE = 'id_rsa'
+NANOPUB_DIR = '.nanopub'
+
+ORCID = '123'
+NAME = 'pietje'
+PUBLISH = False
+
+
+def test_provided_keypair_copied_to_nanopub_dir(tmp_path: Path):
```

Comment:
It feels a bit like the testing code is more complicated (and thus more difficult to maintain) than the code that you're testing. And it only covers a small part of setup_profile.py anyways. What do you think?

Reply:
Yeah I know. I would vote for keeping it, because I think it is very important to check if the cli actually does the stuff it promises to do, so my view on it is that it is not enough testing.  If it becomes a pain in the ass to maintain it we can think of something.

In terms of maintainability I think it wouldn't be that bad. A lot of the code is about creating the right path objects for the rsa keys, which isn't terribly complicated.

Reply:
OK, let's keep it!